src/ui/zcl_abapgit_gui_page_commit.clas.abap

```diff
@@ -479,7 +483,14 @@ CLASS ZCL_ABAPGIT_GUI_PAGE_COMMIT IMPLEMENTATION.
           io_repo   = mo_repo
           io_stage  = mo_stage ).
 
-        MESSAGE 'Commit was successful' TYPE 'S' ##NO_TEXT.
+        lv_repo_log = mo_repo->get_log( ).
+        lt_log_messages = lv_repo_log->get_messages( ).
+        READ TABLE lt_log_messages WITH KEY obj_type = 'SHA1' ASSIGNING <ls_message>.
+        IF sy-subrc = 0.
+          gs_success_log_entry = <ls_message>.
+        ELSE.
+          MESSAGE 'Commit was successful' TYPE 'S' ##NO_TEXT.
+        ENDIF.
```

Comment:
Should not it be one unified approach to show messages ? Html only or message only ?